torch/quantization/quantize.py

```diff
@@ -246,7 +246,8 @@ def quantize(model, run_fn, run_args, mapping=DEFAULT_MODULE_MAPPING):
     """
     model.eval()
     model = prepare(model)
-    run_fn(model, run_args)
+    if run_fn:
+        run_fn(model, run_args)
```

Comment:
if you don't run it how do we get qparams?

Reply:
I have the changes to observer to do that in a seperate diff: https://github.com/pytorch/pytorch/pull/24923.  This is being done to speed up creation of quantized model. Currently if I want to load a quantized model, I have to first call prepare and then convert. Eliminating the eval step speeds the process.

Reply:
I see